releasenotes/notes/fix_log_collection-c6273a2bbdd553e7.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix log collection on Kubernetes distributions using ``cri-o`` like OpenShift when `logs_config.cca_id_ad` is `true`, which is the default value since agent version ``7.40.0``.
```

This feature flag is not exposed to users, so probably best omitted from the changelog unless necessary.

```suggestion
    Fix log collection on Kubernetes distributions using ``cri-o`` like OpenShift, which
    began failing in 7.40.0.
```